rest_server/config.py

```diff
@@ -0,0 +1,66 @@
+"""Config settings."""
+
+import dataclasses as dc
+import logging
+
+import coloredlogs  # type: ignore[import]
+from wipac_dev_tools import from_environment_as_dataclass
+
+LOGGER = logging.getLogger("skydriver")
+
+# --------------------------------------------------------------------------------------
+# Constants
+
+
+@dc.dataclass(frozen=True)
+class EnvConfig:
+    """Environment variables."""
+
+    # pylint:disable=invalid-name
+    AUTH_AUDIENCE: str = "skydriver"
+    AUTH_OPENID_URL: str = ""
+    MONGODB_AUTH_PASS: str = ""  # empty means no authentication required
+    MONGODB_AUTH_USER: str = ""  # None means required to specify
+    MONGODB_HOST: str = "localhost"
+    MONGODB_PORT: int = 27017
+    REST_HOST: str = "localhost"
+    REST_PORT: int = 8080
+    CI_TEST: bool = False
+
+
+ENV = from_environment_as_dataclass(EnvConfig)
```

Comment:
While this pattern of a global env variable is mostly ok, one thing it can't do is changing them during testing.  Something to be aware of if you end up needing to do so down the line.

Reply:
Yeah, that's a good point--a definite downside for testing. The alternative is to make `ENV` a function and re-fetch the env vars, but I think that leads to more problems in the production code since the config state is now mutable.

Reply:
The other thing I've done is only get the env vars when needed (usually when making the server).  That technically means it's mutable if you restart the server, but you only do that in testing.